docs/release-notes/8.9.asciidoc

```diff
@@ -1,6 +1,19 @@
 [[release-notes-header-8.9.0]]
 == 8.9
 
+[discrete]
+[[release-notes-8.9.2]]
+=== 8.9.2
+
+[discrete]
+[[bug-fixes-8.9.2]]
+==== Bug fixes
+
+* Fixes legacy showTopN popover actions ({pull}165099[#165099]).
+* Fixes a blocklist bug that prevented file path entries for Windows and macOS applications from being passed as case insensitive ({pull}164200[#164200]).
+* Modal says "duplicating 0 rules" when you duplicate an individual rule ({pull}163908[#163908]).
```

Comment:
@banderror what should the release note summary for this bug fix be?

Reply:
@nastasha-solomon Could be something like "Fixes a message in the confirmation modal window showed when duplicating a rule"

Reply:
```suggestion
* Fixes a bug in the confirmation message that appears when you're duplicating a single rule ({pull}163908[#163908]).
```